standalone/src/test/java/io/pravega/local/TlsProtocolVersion12Test.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.local;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static io.pravega.local.PravegaSanityTests.testWriteAndReadAnEvent;
+
+public class TlsProtocolVersion12Test {
+
+    @ClassRule
+    public static final PravegaEmulatorResource EMULATOR = PravegaEmulatorResource.builder().tlsEnabled(true).tlsProtocolVersion(new String[] {"TLSv1.2"}).build();
+
+    @Test(timeout = 30000)
+    public void testTlsProtocolVersiontls1_2() throws Exception {
```

Comment:
Do we actually need this test? How long does it take to run? If it takes 10+ seconds, is there a way we can live without having these tests? (you essentially do it twice, here and below).

Reply:
These were added to verify clients communication with Pravega in tls1.2 and tls1.3

Reply:
You still haven't answered my question. These tests are not trivial and usually take a while to run. How long do they take? We also need to factor in whether our build time is going to grow as a result of this change.

Reply:
Ran the test multiple times in isolation, it took ~13-15 seconds for completion. It should take less time when it's run in parallel with other tests.